HTMLtask1/templ1.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>Ellipsis</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
+  <link rel="stylesheet" type="text/css" media="screen" href="templStyle1.css" />
+</head>
+<body>
+  <header class="main-page">
```

Comment:
if you use `header` in most cases you won't need class, just use `header` in your css